### PR TITLE
Api 296: transform behat tests of the PQB of the exports into phpunit tests

### PR DESCRIPTION
**Pull request by ahocquard**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

I've deleted every behat tests that just run a tests through the UI, but without doing anything on the PQB => it should be integration tests with phpunit.

I've merged a lot of behat tests into a single one, by testing more than a filter in the same test. The granularity of the tests should be shifted to the integration tests.
Behat tests just have to assert that the UI PQB filters are correctly passed to the backend.

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | Ok
| Added integration tests           | Ok
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Bundle/ImportExportBundle/tests/integration/Export/AbstractExportTestCase.php [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    {
+        $attribute = $this->get('pim_catalog.factory.attribute')->create();
+        $this->get('pim_catalog.updater.attribute')->update($attribute, $data);
+        $this->get('pim_catalog.saver.attribute')->save($attribute);
+
+        return $attribute;
+    }
+
+    /**
+     * @param array  $data
```
Comment (ahocquard) [inline src/Pim/Bundle/ImportExportBundle/tests/integration/Export/AbstractExportTestCase.php]: same, you will be whipped Alexandre

File: src/Pim/Bundle/ImportExportBundle/tests/integration/Export/AbstractExportTestCase.php [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        $this->get('pim_catalog.updater.product')->update($product, $data);
+        $this->get('pim_catalog.saver.product')->save($product);
+
+        $this->get('akeneo_elasticsearch.client')->refreshIndex();
+
+        return $product;
+    }
+
+    /**
+     * @param array  $data
```
Comment (ahocquard) [inline src/Pim/Bundle/ImportExportBundle/tests/integration/Export/AbstractExportTestCase.php]: double space

File: features/export/product-export-builder/export_products_by_categories.feature [hunk 3]
```diff
@@ -18,6 +18,7 @@ Feature: Export products from any given categories
       | shoes                  | Shoes                     | women                  |
       | jewelry                | Jewelry                   | women                  |
       | men                    | Men                       | clothing_shoes_jewelry |
+      | 1234                   | 1234                      | default |
```
Comment (ahocquard) [inline features/export/product-export-builder/export_products_by_categories.feature]: align |

File: src/Pim/Bundle/ConnectorBundle/tests/integration/Export/ProductQueryBuilder/ExportProductsByBooleanIntegration.php [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        ]);
+
+        $this->createproduct('product_without_boolean', [
+            'values'     => [
+                'a_number_float' => [['data' => '20.09', 'locale' => null, 'scope' => null]],
+            ]
+        ]);
+    }
+
+    public function testProductExportWithBooleanFilterEqualsFalse()
```
Comment (skeleton) [inline src/Pim/Bundle/ConnectorBundle/tests/integration/Export/ProductQueryBuilder/ExportProductsByBooleanIntegration.php]: EqualsTrue

**Comment by ahocquard** [comment]
As seen with @skeleton IRL, we will not cover every operators for now, because it's the same as testing the PQB, which is already well covered.

**Pull request merged**